include/ebpf_extension.h

```diff
@@ -15,18 +15,63 @@ typedef struct _ebpf_extension_dispatch_table
     _Field_size_(count) _ebpf_extension_dispatch_function function[1];
 } ebpf_extension_dispatch_table_t;
 
+/**
+ * @brief Invoke the eBPF program.
+ *
+ * @param[in] extension_client_binding_context The context provided by the extension client when the binding was
+ * created.
+ * @param[in,out] program_context The context for this invocation of the eBPF program.
+ * @param[out] result The result of the eBPF program.
+ *
+ * @retval EBPF_SUCCESS if successful or an appropriate error code.
```

Comment:
The text following the error code should be phrased as a complete sentence.  "if successful or an appropriate error code" is not a complete sentence, nor does "an appropriate error code" apply to the EBPF_SUCCESS case.

Need changes throughout this file.